lua/nvim-treesitter/textobjects/attach.lua

```diff
@@ -29,7 +33,13 @@ function M.make_attach(normal_mode_functions, submodule)
             .. "("
             .. config_query_str
             .. ")<CR>"
-          api.nvim_buf_set_keymap(bufnr, "n", mapping, cmd, { silent = true, noremap = true })
```

Comment:
if desired I could allow mapping to be also be optionally a table and then access mapping.desc

Reply:
You already did right? That is just perfect

Reply:
No, not yet. At the moment it will just generate a description for all commands but you can't set your own. Is setting your own description still needed?

Reply:
Yes, setting my own description is why I opened the issue in the first place. I can see a lot of people not being happy with the auto generated ones.